alicloud/resource_alicloud_ssl_vpn_client_cert.go

```diff
@@ -37,6 +37,26 @@ func resourceAliyunSslVpnClientCert() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"ca_cert": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+			"client_cert": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+			"client_key": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+			"client_config": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
```

Comment:
Please add docs for above four new parameters.

Reply:
How do I add docs? Do I just manually add it, or are they generated somehow?